Dockerfile

```diff
@@ -63,6 +64,8 @@ RUN helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx && \
 # pulling `terra-app-setup` locally and add cert files to the chart.
 # Leonardo will install the chart from local version.
 # We are also caching charts so they are not downloaded with every helm-install
+COPY ./relay-listener-0.1.0.tgz /leonardo
```

Comment:
TODO: remove

Reply:
didnt' realize relay is installed as it's own chart

Reply:
It used to be part of the setup chart, but we're breaking it into a dedicated chart in terra-helmfile. See https://github.com/broadinstitute/terra-helmfile/pull/4411